### PR TITLE
Make all non-web tooltips lowercase

### PR DESCRIPTION
**Pull request by revelsix**
Step 1 and 2 of 3 in trying (again) to fix the inconsistent tooltip capitalization throughout the game. The other step requires changing localization files in osu-web (outlined in #21764). This change assumes all tooltip text (excluding mods and countries) should be lowercase, including tooltips that use multiple sentences, e.g.: 
image

### CODE REVIEW

**Comment by Joehuu** [comment]
I somewhat disagree with this direction of "styling" every tooltip to be *mostly all* lowercase. Personally, I prefer normalizing everything to sentence-case, even if it is the minority right now.

Apart from the multiple-sentence styling which doesn't sit right in my opinion (makes sentences harder to read), what happens to proper nouns? 
- There's a string here that has lowercase `windows`. Should it stay lowercase?
- I also saw the web-side PR. Should `show on GitHub` be `show on github` if the above stays?

**Review by Joehuu — CHANGES_REQUESTED** [review]
As above and just to mark this PR with changes requested.

File: osu.Game/Localisation/MouseSettingsStrings.cs [hunk 1]
```diff
@@ -15,19 +15,19 @@ public static class MouseSettingsStrings
...
         /// "High precision mouse"
         /// </summary>
         public static LocalisableString HighPrecisionMouse => new TranslatableString(getKey(@"high_precision_mouse"), @"High precision mouse");
 
         /// <summary>
-        /// "Attempts to bypass any operation system mouse acceleration. On windows, this is equivalent to what used to be known as &quot;Raw Input&quot;."
+        /// "attempts to bypass any operating system mouse acceleration. on windows, this is equivalent to what used to be known as &quot;raw input&quot;."
         /// </summary>
-        public static LocalisableString HighPrecisionMouseTooltip => new TranslatableString(getKey(@"high_precision_mouse_tooltip"), @"Attempts to bypass any operation system mouse acceleration. On windows, this is equivalent to what used to be known as ""Raw Input"".");
+        public static LocalisableString HighPrecisionMouseTooltip => new TranslatableString(getKey(@"high_precision_mouse_tooltip"), @"attempts to bypass any operating system mouse acceleration. on windows, this is equivalent to what used to be known as ""raw input"".");
```
Comment (r00ster91) [inline osu.Game/Localisation/MouseSettingsStrings.cs:30]:
```suggestion
        public static LocalisableString HighPrecisionMouseTooltip => new TranslatableString(getKey(@"high_precision_mouse_tooltip"), @"attempts to bypass any operating system mouse acceleration. on Windows, this is equivalent to what used to be known as ""raw input"".");
```
What about proper nouns?

**Comment by Joehuu** [comment]
After looking at this again, I think we should use sentence-case. My reasons above are still the same, plus no other website/app is using lowercase. The 2+ sentence tooltip problem is unavoidable, and we shouldn't have case exceptions to sentences and non-sentences as that gets complicated.

@ppy/team-client thoughts before I attempt normalising everything to sentence-case?

**Comment by Joehuu** [comment]
Closing as incorrect direction.

**Pull request closed without merging**